### ISSUE TITLE
[Webcompat] Login issues on https://store.ubi.com/us/account-connection/?lang=en_US

### ISSUE THREAD

**Issue by ryanbr** [brave/brave-browser#9285]
## Description 
Unable to login on `https://store.ubi.com/us/account-connection/?lang=en_US` 

## Steps to Reproduce

   1. Visit `https://store.ubi.com/us/account-connection/?lang=en_US`
   2. Click on the `Top Right person Avatar` to login to website.
   3. Generates error message

## Actual result:
![ubi](https://user-images.githubusercontent.com/1659004/79519594-6b394d80-80a8-11ea-9ae0-a068c4d68564.png)

## Expected result:
![ubi2](https://user-images.githubusercontent.com/1659004/79519629-7e4c1d80-80a8-11ea-938f-235a93cf8bb7.png)
Login to show up.

## Reproduces how often: 
Easily

## Brave version (brave://version info)
`Version 1.7.92 Chromium: 80.0.3987.163 (Official Build) (64-bit)`

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the dev channel?
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? Yes

## Miscellaneous Information:

All cookies are needed. `connect.ubisoft.com`  Showing Local storage in the settings.

**Comment by ryanbr** [comment]
Needing a whitelist `ubisoft.com` `connect.ubisoft.com` for `ubi.com`

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.11.75 Chromium: 83.0.4103.116 (Official Build) dev (64-bit)
-- | --
Revision | 8f0c18b4dca9b6699eb629be0f51810c24fb6428-refs/branch-heads/4103@{#716}
OS | macOS Version 10.14.6 (Build 18G3020)
```

* Verified STR from description.

* Using default shields, reproduced the issue using `1.10.97`:
<img width="607" alt="Screen Shot 2020-06-24 at 9 42 03 AM" src="https://user-images.githubusercontent.com/28145373/85570304-3d6d0980-b601-11ea-962d-f777a8185b55.png">

* Upgraded to 1.11.x and confirmed I was able to get the login screen as expected. Confirmed able to login as well:
<img width="610" alt="Screen Shot 2020-06-24 at 9 42 44 AM" src="https://user-images.githubusercontent.com/28145373/85570411-537aca00-b601-11ea-85e3-71fcb9578362.png">

* Using a clean profile on 1.11.x with default shields, confirmed able to get the login screen and login as well.
